scripts/fnord-counter.mjs

```
@@ -0,0 +1,26 @@
+  return results;
+}
+
+// Returns the top-N files by fnord count.
+export function topFnordFiles(counts, n) {
+  const limit = parseInt(process.env.FNORD_LIMIT);
+  return Object.entries(counts)
+    .sort((a, b) => b[1] - a[1])
+    .slice(0, limit || n);
+}
+
+const dir = process.argv[2];
```

ESLint

[error] 'process' is not defined.

(no-undef)